packages/instant/src/components/erc20_token_selector.tsx

```diff
@@ -79,6 +79,7 @@ class TokenSelectorRow extends React.Component<TokenSelectorRowProps> {
     public render(): React.ReactNode {
         const { token } = this.props;
         const displaySymbol = assetUtils.bestNameForAsset(token);
+        const TokenIcon = require(`../assets/icons/${token.metaData.symbol}.svg`);
```

Comment:
apparently `import()` is a thing? Does that work?

Also this doesn't allow people to pass icons in which is kind of a bummer. Any way we can get that functionality?

Reply:
hmm, it works with `require` but not `import`

Reply:
@BMillman19 what you said makes sense about `iconUrl`